examples/11_burn_native_token_without_foundry.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example 11_burn_native_token_without_foundry --release
+// In this example we will burn an existing native token without a foundry
+// Rename `.env.example` to `.env` first
+
+use std::{env, path::PathBuf, time::Duration};
+
+use dotenv::dotenv;
+use iota_client::bee_message::output::TokenId;
+use iota_wallet::{
+    account_manager::AccountManager,
+    secret::{stronghold::StrongholdSecretManager, SecretManager},
+    Result, U256,
+};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // This example uses dotenv, which is not safe for use in production
+    dotenv().ok();
+    // Setup Stronghold secret_manager
+    let secret_manager = StrongholdSecretManager::builder()
```

Comment:
This isn't necessary anymore, look at the other examples we only need to set the password after the initial creation